sdgym/synthesizers/uniform.py

```diff
@@ -19,7 +19,8 @@ def _get_trained_synthesizer(self, real_data, metadata):
         hyper_transformer.detect_initial_config(real_data)
         supported_sdtypes = hyper_transformer._get_supported_sdtypes()
         config = {}
-        for column_name, column in metadata.columns.items():
+        table_name = metadata._get_single_table_name()
+        for column_name, column in metadata.tables[table_name].columns.items():
```

Comment:
same thing here. I think it's best to avoid underscore methods, even in our own ecosystem as they are meant to suggest that they might change or be deleted at any point in time